ci/buildspec.yml

```diff
@@ -0,0 +1,30 @@
+# Build Spec for AWS CodeBuild CI
+
+version: 0.2
+
+phases:
+  install:
+    commands:
+      - apt-get update
+      - apt-get install -y curl
+      - apt-get install -y libgfortran3
+      - pip install pip -U
+      - pip install flask
+      #- pip install -U mxnet
```

Comment:
Why is this commented out?

Reply:
We don't use these dependencies during build.